mw/notification.d.ts

```diff
@@ -40,6 +36,15 @@ declare global {
                 pause: () => void;
                 resume: () => void;
                 close: () => void;
+                $notification: JQuery;
+                autoHideSeconds: number;
+                isOpen: boolean;
+                isPaused: boolean;
+                options: Partial<typeof notification.defaults>;
+                timeout: {
+                    set: typeof setTimeout;
+                    clear: typeof clearTimeout;
+                }
```

Comment:
Let's define this as a `Notification` interface (to be reused in mw.notify's return type).

Also, message and start() are missing.